demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -852,34 +853,32 @@ def filter_changed_files(self, files_string, tag='master', print_ignored_files=F
                     not (file_path.endswith('_test.py') or file_path.endswith('.Tests.ps1')):
                 # naming convention - code file and yml file in packages must have same name.
                 file_path = os.path.splitext(file_path)[0] + '.yml'
-
             # ignore changes in JS files and unit test files.
             elif file_path.endswith('.js') or file_path.endswith('.py') or file_path.endswith('.ps1'):
                 self.ignored_files.add(file_path)
```

Comment:
if you are here - we might want to add this here as well:
```
if print_ignored_files:
        click.secho('Ignoring file path: {}'.format(file_path), fg="yellow")
```

Reply:
Added in a few other places as well.